docs/howto.rst

```diff
@@ -52,7 +52,25 @@ The ``auth`` argument should be a tuple of the form ``('username', 'password')``
 
 Full example: :download:`basic_auth.py <examples/basic_auth.py>`
 
+HTTP Digest authentication is supported by passing an instance of
+:py:class:`treq.auth.HTTPDigestAuth` class with ``auth`` keyword argument to any of
```

Comment:
Maybe this would be better as "HTTP Digest authentication is supported by passing an instance of :py:class:`treq.auth.HTTPDigestAuth` to any of the request functions by using the ``auth`` keyword argument."